docker/examples/docker-compose-rabbitmq.yml

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2015-2023 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+# This file uses the version 2 docker-compose file format, described here:
+# https://docs.docker.com/compose/compose-file/#version-2
+#
+# It extends the default configuration from docker-compose.yml to add a test
+# rabbitmq server, which is used as a span transport.
+
+version: '2.4'
+
+services:
+  rabbitmq:
```

Comment:
before, doing this would be tricky because we don't lazy create the queue in zipkin's java code. Since this image has a default queue, it is possible to make a simple example now.